pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -0,0 +1,21 @@
+package bootstrap
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// verifyBootstrapInPlace validate the number of control plane replica is one and that installation disk is set
+func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
+	if *installConfig.ControlPlane.Replicas != 1 {
+		return errors.Errorf("bootstrap in place require single control plane replica, current value: %d", *installConfig.ControlPlane.Replicas)
+	}
+	if installConfig.BootstrapInPlace == nil {
+		return errors.Errorf("Missing bootstrap in place configuration")
```

Comment:
nit: [error strings should not be capitalized][1].

[1]: https://github.com/golang/go/wiki/CodeReviewComments#error-strings